pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -115,6 +117,32 @@ func (c *Client) InstanceExistsInCluster(operatorName, namespace, version, insta
 	return true, nil
 }
 
+// GetInstance queries kubernetes api for instance of given name in given namespace
+// returns error for all other errors that not found, not found is treated as result being 'nil, nil'
```

Comment:
comment is hard to parse... I understand what is meant... 
```suggestion
// returns error for error conditions.  instance not found is not considered an error and will result in 'nil, nil'
```